src/content/docs/import/csv.md

```diff
@@ -187,91 +187,3 @@ The possible configurations for different configurations are:
-```
-Output:
-```
-┌──────────┬
-│ count(*) │
-│ UINT64   │
-├──────────┼
-│ 1        │
-└──────────┴
-```
-
-### Warnings Table: Inspecting skipped rows
-Information about malformed lines are kept in a system-level Warnings table.
-Warnings table is a connection-level table that accumulates warnings that are generated during COPY statements during the
-lifetime of a database connection. You can inspect the contents of this table, e.g., to see the warnings
-about the malformed rows, or clear this table. 
-
-For example, to see the warning messages generated about the malformed line in our 'user.csv' import above,
-you can run the [`SHOW_WARNINGS`](/cypher/query-clauses/call#show_warnings) function after the `COPY FROM` statement:
-
-```cypher
-CALL show_warnings() RETURN *;
-```
-
-Output:
-```
-┌──────────┬─────────────────────────────────────────────────────────────────────────────┬─────────────┬─────────────┬────────────────────────┐
-│ query_id │ message                                                                     │ file_path   │ line_number │ skipped_line_or_record │
-│ UINT64   │ STRING                                                                      │ STRING      │ UINT64      │ STRING                 │
-├──────────┼─────────────────────────────────────────────────────────────────────────────┼─────────────┼─────────────┼────────────────────────┤
-│ 1        │ Conversion exception: Cast failed. Could not convert "2147483650" to INT32. │ vPerson.csv │ 2           │ 2,2147483650           │
-└──────────┴─────────────────────────────────────────────────────────────────────────────┴─────────────┴─────────────┴────────────────────────┘
-```
-
-At any point in time you can also call the [`CLEAR_WARNINGS`](/cypher/query-clauses/call#clear_warnings) function to clear the Warnings table.
-
-```cypher
-CALL clear_warnings();
-```
-
-After clearing the warnings, the Warnings table will be empty.
-
-```cypher
-CALL show_warnings() RETURN *;
-```
-
-Output:
-```
-┌──────────┬─────────────────────────────────────────────────────────────────────────────┬─────────────┬─────────────┬────────────────────────┐
-│ query_id │ message                                                                     │ file_path   │ line_number │ skipped_line_or_record │
-│ UINT64   │ STRING                                                                      │ STRING      │ UINT64      │ STRING                 │
-└──────────┴─────────────────────────────────────────────────────────────────────────────┴─────────────┴─────────────┴────────────────────────┘
-```
-
-By default, Kùzu stores a limited number of warnings per connection, determined by the [`warning_limit`](/cypher/configuration#connection-configuration) connection configuration parameter.
-You can change this configuration as follows (see the [Connection Configuration](/cypher/configuration#connection-configuration) section for more details):
-```cypher
-CALL warning_limit=1024;
-```
```

Comment:
Moved to separate page (see PR description)

Reply:
Why does this need to be moved to a separate page @royi-luo?

Reply:
It applies to multiple input source types so I thought it didn't make sense for it to be under CSV. Do you have any ideas on where else it can be put?

Reply:
Maybe it can be a separate page but not be in the side bar?